app/services/encryptor_service.rb

```diff
@@ -1,7 +1,5 @@
 class EncryptorService
-  def initialize(
-    key = Rails.application.secrets.secret_key_base.bytes[0..31].pack('c' * 32)
-  )
+  def initialize(key = Settings.secret_key_base.bytes[0..31].pack('c' * 32))
     @crypt = ActiveSupport::MessageEncryptor.new(key, digest: 'base64_urlsafe')
```

Comment:
@kaisersakhi 

This(`@crypt`) when used with `decrypt_and_verify(value, **options)` returns the response keys as a string. Previously it was returning keys as a symbol

This change is causing Google `OAuth` login to fail, and I think this would fail for other OAuth logins as well.

```
NoMethodError (undefined method `end_with?' for nil:NilClass):
```

It is returning keys as strings because of this new change:

https://github.com/pupilfirst/pupilfirst/blob/1cb463aa11ac662c3443138a373636fcb9b88d82/config/initializers/new_framework_defaults_7_1.rb#L141

We can fix this one of the two ways:

1. Convert the response keys to symbols 

```rb
.transform_keys(&:to_sym)
```

2. Use strings as keys 

```rb
data["session_id"]
```